support-docs/examples.md

```diff
@@ -1935,3 +1797,223 @@ PMIC Example - Limited pins
         }
     ]
 }
+```
+
+Sensor Example - Limited pins 
+
+```json     
+{
+    "componentID": {
+        "partType": "accelerometer",
+        "manufacturer": "ACME Components",
+        "componentName": "abc100",
+        "orderableMPN": "abc100,456",
+        "sourceDatasheetID": {
+            "publishedDate": "2022-03-23",
+            "datasheetURI": "www.acmecomponents.com/abc100datasheet"
+        },
+        "digitalDatasheetID": {
+            "publishedDate": "2023-03-23",
+            "guid": "3e4cd9de-657a-41ae-902e-beca95aff51d"
+        },
+        "status": "active"
+    },
+    "accelerationRanges": 8,
+    "accelerationSensitivity": {
```

Comment:
it occurs to me that conditionalProperty (or unit) needs a way to specify things that aren't listed in the typical list of units. 

We can talk about this in the meeting tomorrow if you want. I might suggest adding another field to conditionalProperty so we have:

"value": "definitions.json#/unit"
"customValue": "definitions.json#/value"

Reply:
Sounds good